si2.bll/Services/InstitutionService.cs

```diff
@@ -79,8 +79,6 @@ public async Task<PagedList<InstitutionDto>> GetInstitutionsAsync(InstitutionRes
                 .GetAllComplete()
                 .Where(c => resourceParameters.ParentId == null || c.ParentId == resourceParameters.ParentId);
 
-            if (institutionEntities.Count() > 1)
```

Comment:
Why did you remove this validation from here? we need to keep this validation here